File: FunProg/code_1.py
import math
n=786432
count=0
x=1

sqr=math.sqrt(n)
while(sqr!=int(sqr)):
    n=n*x
    sqr=math.sqrt(n)
    x=x+1
    count=count+1

print(count)
print(sqr)

File: FunProg/code_1.c
#include <stdio.h>
#include <math.h>
int main(){
  int n=786432,count=0,x=1;
  float sqr;
  sqr=sqrt(n);
  while(sqr!=(int)sqr){
    n=n*x;
    sqr=sqrt(n);
    x++;
    count++;
  }
printf("%d\n",count);
printf("%d\n",sqr);
  return 0;
}
